packages/geoprocessing/src/toolbox/overlapFeatures.ts

```diff
@@ -20,6 +20,8 @@ interface OverlapFeatureOptions {
   /** If sketch collection, will include its child sketch metrics in addition to collection metrics, defaults to true */
   includeChildMetrics?: boolean;
   sumProperty?: string;
+  /** Simplifies results to 6 significant digits to avoid floating pt arithmetric differences, defaults to false */
+  simplifyPrecision?: boolean;
```

Comment:
Consider using the word `truncate` instead of simplify/simplifyPrecision.  I may have used the word simplify previously, I would say that was before I was aware of the term truncate.  Personally, simplify makes me think it's modifying geometries to be simpler, less coordinates.  The word `precision` may not be needed unless you are offering to let the user specify the precision.

Reply:
toPrecision() doesn't truncate, it rounds to significant digits. I could use round though.

toFixed() truncates, but it doesn't use significant digits, it counts digits past the decimal point.

Reply:
Updated to `truncate?: boolean;`